connect/api/src/main/java/org/apache/kafka/connect/data/SchemaProjector.java

```diff
@@ -109,16 +109,14 @@ private static Object projectStruct(Schema source, Struct sourceStruct, Schema t
                     Object targetFieldValue = project(sourceField.schema(), sourceFieldValue, targetField.schema());
                     targetStruct.put(fieldName, targetFieldValue);
                 } catch (SchemaProjectorException e) {
-                    throw new SchemaProjectorException("Error projecting " + sourceField.name(), e);
+                    throw new SchemaProjectorException("Error projecting " + fieldName, e);
```

Comment:
Source field name was given before, and now it's the target field name that is supplied in the Exception. Would we want both? (something like `"Error projecting " + fieldName + " to " + sourceField.name()"` or is this unneeded?